apps/service_define/dev_scripts/send_events.exs

```diff
@@ -0,0 +1,20 @@
+id = "My-id"
+
+extract_event =
+    Extract.new!(
+    id: "extract-1",
+    dataset_id: id,
+    subset_id: "default",
+    destination: "success",
+    dictionary: [
+        Dictionary.Type.String.new!(name: "letter")
+    ],
+    steps: [
+        Extract.Http.Get.new!(
+        url: "http://localhost/file.csv",
+        headers: %{"content-length" => "5"}
```

Comment:
Why do we need to tell the content length on the request as a header?

Reply:
We don't, it was just what we thought of first. Would you prefer we use something else?